modules/sdk/gradle-plugins-source-formatter/README.markdown

```diff
@@ -44,13 +40,11 @@ Name | Depends On | Type | Description
 ---- | ---------- | ---- | -----------
 `formatSource` | - | [`FormatSourceTask`](#formatsourcetask) | Runs Liferay Source Formatter to format the project files.
 
-## Task Types
-
-### <a name="formatsourcetask"></a>`FormatSourceTask`
```

Comment:
The anchor is not necessary because Github already provides an anchor with the `##` heading. The way you linked it is still correct for Github's automatic anchor, so I left it there. Let me know if I'm missing something :smiley: